### ISSUE TITLE
Dashboard / Overview pages doesn't work

### ISSUE THREAD

**Issue by it4need** [StydeNet/enlighten#12]
Without a failing test, I cannot see any tests in the dashboard. 

If I make one test failed, that I get an url like in the screenshot from the terminal 
<img width="895" alt="Bildschirmfoto 2020-10-16 um 02 40 16" src="https://user-images.githubusercontent.com/2446843/96199928-f1035c00-0f58-11eb-9b13-f437e4ffa3e1.png">

When I go to the site, I can see everything - just fine...
<img width="1654" alt="Bildschirmfoto 2020-10-16 um 02 41 07" src="https://user-images.githubusercontent.com/2446843/96200173-8acb0900-0f59-11eb-8b19-44b2980393cc.png">

But when I click on any sidebar item, I will get an empty result (where I think the tests should be listed).
<img width="1655" alt="Bildschirmfoto 2020-10-16 um 02 41 20" src="https://user-images.githubusercontent.com/2446843/96200204-a0403300-0f59-11eb-8c70-f5a37b5264c1.png">

Same for the /enlight/ dashboard after clicking "View"
<img width="1656" alt="Bildschirmfoto 2020-10-16 um 02 41 34" src="https://user-images.githubusercontent.com/2446843/96200214-a9c99b00-0f59-11eb-989b-dcdd73857c62.png">

```json
"require": {
        "php": "^7.4",
        "ext-json": "*",
        "fideloper/proxy": "^4.2",
        "fruitcake/laravel-cors": "^1.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "laravel/framework": "^8.0",
        "laravel/legacy-factories": "^1.0",
        "laravel/tinker": "^2.0",
        "nwidart/laravel-modules": "^7.1",
        "wikimedia/composer-merge-plugin": "^1.4"
    },
```

**Comment by jeffochoa** [comment]
This is ready to go on the next release. 

https://github.com/StydeNet/enlighten/commit/099367c680857e134f0a197156d6e6e23578cb99

You can use the `main` branch in the meantime. Please refresh the migrations after pulling these changes.

Thanks!

**Comment by sileence** [comment]
@it4need this should be now fixed in `v0.2.8`, if it's not please contact me on Discord (https://discord.gg/hNCV6p) as I was unable to reproduce the error in my local environment.

Thanks for trying our component :)

**Comment by it4need** [comment]
At first, thanks for the fast reply. 

These pages (`/enlighten/run/1/site`) are looking good now. But the dashboard (`/enlighten/`) is now unaccessable, resulting in a `403 Forbidden`. 

On top of that, only for your information - after I did the update to `v.0.2.8` I can't just do `php artisan migrate:fresh`. It didn't clear out the `ayda-api-enlighten` database (as I defined it in the config). I did it manually and migrated the database afterwords - that was the solution for this problem.

**Config:**
```php
        'enlighten' => [
            'driver'         => 'mysql',
            'url'            => env('DATABASE_URL'),
            'host'           => env('DB_HOST', '127.0.0.1'),
            'port'           => env('DB_PORT', '3306'),
            'database'       => 'ayda-api-enlighten',
            'username'       => env('DB_USERNAME', 'forge'),
            'password'       => env('DB_PASSWORD', ''),
            'unix_socket'    => env('DB_SOCKET', ''),
            'charset'        => 'utf8mb4',
            'collation'      => 'utf8mb4_unicode_ci',
            'prefix'         => '',
            'prefix_indexes' => true,
            'strict'         => true,
            'engine'         => null,
            'options'        => extension_loaded('pdo_mysql') ? array_filter([
                PDO::MYSQL_ATTR_SSL_CA => env('MYSQL_ATTR_SSL_CA'),
            ]) : [],
```
